ethash/src/compute.rs

```diff
@@ -141,6 +145,49 @@ impl Light {
 	}
 }
 
+pub struct EthCompute {
+	prev_epoch: u64,
```

Comment:
This might be a matter of style. I like having to use the object mutably, and there is precedent in Rust for this design (e.g. Iterators don't use Cell for internal state). I can change it if you think it will harm ergonomics though.